lib/form_data.js

```diff
@@ -496,6 +497,16 @@ FormData.prototype._error = function(err) {
   }
 };
 
-FormData.prototype.toString = function () {
-  return '[object FormData]';
-};
+if (hasToStringTag()) {
+  Object.defineProperty(FormData.prototype, Symbol.toStringTag, {
+    __proto__: null,
+    configurable: true,
+    enumerable: false,
+    writable: false,
+    value: 'FormData',
+  });
+} else {
+  FormData.prototype.toString = function () {
+    return '[object FormData]';
+  };
```

Comment:
the toString is not the same as toStringTag, and shouldn't be defined at all - separately, i have a package for this already, es-set-tostringtag